.gitlab-ci.yml

```diff
@@ -141,15 +141,11 @@ variables:
       - $RELEASE_VERSION_6 == "nightly"
       - $RELEASE_VERSION_7 == "nightly-a7"
 
-# run job only when triggered on nightly AND when it's a merge to master
-.run_when_triggered_on_nightly_or_master: &run_when_triggered_on_nightly_or_master
-  only:
+# run when not triggered (for jobs we don't want to run on release pipelines)
+.run_when_not_triggered: &run_when_not_triggered
```

Comment:
I'm open to renaming it, but my intent with this name was to distinguish it from other mixins that define when to run jobs, since this one is using an `except` clause which defines when _not_ to run. Technically it could even be used together with a `run_when_` job and both would take effect.

Reply:
What we could do is match more closely the job definitions:
- `run_only_when/on_*` for `when:only` anchors
- `run_except_when/on_*` for `when:except` anchors
to make all our anchor definitions consistent. But maybe not in this PR as that would need a bigger renaming change and we're very close to the freeze.

Reply:
Last commit uses `run_only_when_*` for all `only` and `skip_when_*` for all `except`.

We should soon change these to use rules instead of only/except.